ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master.yaml

```diff
@@ -188,6 +188,21 @@ tests:
       TEST_TYPE: upgrade-conformance
     workflow: openshift-upgrade-aws-ovn
   timeout: 5h30m0s
+- as: e2e-aws-ovn-ipsec-upgrade
```

Comment:
~this is minor upgrade ? master to master+PR?~
I assume this is used for both major and minor upgrades from config standpoint?